blog/2023-01-18-introducing-netdata-paid-plans/index.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: introducing-netdata-paid-subscriptions
+title: Introducing Netdata Paid Subscriptions
+description: All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for businesses of different sizes and with different monitoring requirements.
+authors: team
+tags: [pricing, free-plan, paid-subscriptions]
+keywords: [pricing, free plan, paid subscriptions, monitoring, troubleshoot]
+image: ./img/subscriptions.png
+
+---
+
+Read more about Netdata introducing paid subscriptions. All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for businesses of different sizes and with different monitoring requirements.
+
+<!--truncate-->
+**Hello Netdata community**,
+
+We are excited to announce that we are introducing new paid subscriptions to our SaaS offering. At Netdata, we believe in providing free and unrestricted access to high-quality monitoring solutions, and our commitment to this principle will not change. We will continue to offer our free SaaS offering and Open Source Agent, which features unlimited nodes and users, unlimited metrics, and retention, providing real-time, high-fidelity, out-of-the-box infrastructure monitoring for packaged applications, containers, and operating systems.
+
```

Comment:
Maybe add something along these lines

"Netdata's vision is to make monitoring simple and accessible for everyone, democratizing monitoring for businesses of all sizes. To help fulfill this vision, we have decided to launch a paid subscription for pro and business users."